ci/README.md

```diff
@@ -71,3 +71,22 @@ The `./ci/run_envoy_docker.sh './ci/do_ci.sh <TARGET>'` targets are:
 * `bazel.tsan` &mdash; build and run tests under `-c dbg --config=clang-tsan` with clang-5.0.
 * `check_format`&mdash; run `clang-format` 5.0 and `buildifier` on entire source tree.
 * `fix_format`&mdash; run and enforce `clang-format` 5.0 and `buildifier` on entire source tree.
+
+# Testing changes to docker as a developer
```

Comment:
nit: I would s/docker/"build image" (or something like that) throughout the change. I think that would be more clear. Thanks for updating the docs. This is great.